packages/paste-core/components/combobox/stories/index.stories.tsx

```diff
@@ -375,7 +377,7 @@ storiesOf('Components|Combobox', module)
               setValue(inputValue);
             }
           }}
-          itemToString={item => (item ? item.label : null)}
+          itemToString={(item: any) => (item ? item.label : null)}
```

Comment:
I don't think these should be `any`. Maybe `{label: string}` instead? Like if it is a number this would break.

Reply:
Yeah, because they're just stories I wasn't being too fussy with typing the returned data we feed the example. I was just trying to get rid of the red underline. But I can change them to strings